deepchecks/vision/checks/performance/confusion_matrix.py

```diff
@@ -143,18 +148,11 @@ def compute(self, context: Context, dataset_kind: DatasetKind = None) -> CheckRe
                 x.append(it)
                 y.append(it)
             else:
-                x.append('No overlapping prediction')
-                y.append('No overlapping label')
+                x.append('No overlapping')
```

Comment:
why removed prediction/label?

Reply:
agree

Reply:
it makes the confusion matrix much smaller